targets/targets.json

```diff
@@ -2052,28 +2038,24 @@
         "supported_form_factors": ["ARDUINO"],
         "release_versions": ["5"],
         "config": {
-            "stdio_uart_tx": {
-                "help": "Value: D8(default) or D1",
-                "value": "D8"
-            },
-            "stdio_uart_rx": {
-                "help": "Value: D2(default) or D0",
-                "value": "D2"
-            }
+            "stdio_uart_tx_help": { "help": "Value: D8(default) or D1" },
```

Comment:
can you split this to multiple lines as it was? (similar to other lines)

Reply:
OK
Done!